br/pkg/lightning/backend/local/local.go

```diff
@@ -1789,14 +1789,18 @@ func (local *local) isIngestRetryable(
 		if newRegion != nil {
 			retryTy = retryWrite
 		}
-		return retryTy, newRegion, errors.Errorf("epoch not match: %s", errPb.GetMessage())
+		return retryTy, newRegion, errors.Annotate(common.ErrKVEpochNotMatch, errPb.GetMessage())
 	case strings.Contains(errPb.Message, "raft: proposal dropped"):
 		// TODO: we should change 'Raft raft: proposal dropped' to a error type like 'NotLeader'
 		newRegion, err = getRegion()
 		if err != nil {
 			return retryNone, nil, errors.Trace(err)
 		}
 		return retryWrite, newRegion, errors.New(errPb.GetMessage())
+	case errPb.ServerIsBusy != nil:
+		return retryNone, nil, errors.Annotate(common.ErrKVServerIsBusy, errPb.GetMessage())
+	case errPb.RegionNotFound != nil:
+		return retryNone, nil, errors.Annotate(common.ErrKVRegionNotFound, errPb.GetMessage())
```

Comment:
Does it mean we don't retry for `ServerIsBusy` and `RegionNotFound`?

Reply:
those two are copies of below
```golang
return retryNone, nil, errors.Errorf("non-retryable error: %s", resp.GetError().GetMessage())
```
just to give them a unique error code

Reply:
> Does it mean we don't retry for `ServerIsBusy` and `RegionNotFound`?

will retry

Reply:
Seems inside this function there's retry level of "Write+Injest", "Injest", and in the caller there's "ScanRegion+Write+Injest". Maybe we can refine them later